src/devices/Buzzer/samples/Buzzer.Samples.csproj

```diff
@@ -6,8 +6,8 @@
   </PropertyGroup>
 
   <ItemGroup>
-  	<ProjectReference Include="$(MSBuildThisFileDirectory)../Buzzer.csproj" />
-  	<ProjectReference Include="$(MainLibraryPath)System.Device.Gpio.csproj">
+      <ProjectReference Include="$(MSBuildThisFileDirectory)../Buzzer.csproj" />
```

Comment:
wow, did style cop check that?

Reply:
Nope, I did a search and replace for tabs 😄

Reply:
works too 😄